packages/elastic-datemath/tsconfig.json

```diff
@@ -1,9 +1,17 @@
 {
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "tsBuildInfoFile": "../../build/tsbuildinfo/packages/elastic-datemath"
+    "incremental": false,
```

Comment:
I know we have gone back-and-forth on if we should use incremental, and it's kind of moot with Bazel, but did it need disabled here as part of this fix?

Reply:
bazel will have it enabled by default. I just decided to remove it here as a realised if for some reason I manually deleted the `target` folder, it won't be recreated upon a build with the incremental on.